app/views/rejected_notification_errors.scala.html

```diff
@@ -16,56 +16,68 @@
 
 @import models.declaration.submissions.Submission
 @import services.model.RejectionReason
-@import views.Title
 @import config.AppConfig
+@import views.{BackButton, Title}
+@import views.html.components.gds._
+@import uk.gov.hmrc.govukfrontend.views.html.components._
 
-@this(main_template: views.html.main_template, appConfig: AppConfig)
+@this(
+        govukLayout: gdsMainTemplate,
+        pageTitle: pageTitle,
+        sectionHeader: sectionHeader,
+        link: link,
+        govukTable: GovukTable,
+        govukSummaryList : GovukSummaryList,
+        appConfig: AppConfig
```

Comment:
Probably worth to consider in the Future about the new config class for errors descriptions, to not inject the whole appConfig to the view, but it's not something that must be done now.